<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributeBrowserWidget</class>
 <widget class="QWidget" name="AttributeBrowserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Node</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="node_id_label">
         <property name="text">
          <string>(ID, type)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="node_cb"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Node attributes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="node_attrib_tw"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="new_node_attrib_pb">
         <property name="text">
          <string>New attrib</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="del_node_attrib_pb">
         <property name="text">
          <string>Del Atrrib</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_node_pb">
         <property name="text">
          <string>Save Node</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="edges_groupbox">
       <property name="title">
        <string>Edges</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Edge</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="edge_id_label">
              <property name="text">
               <string>(ID, type)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="edge_cb"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Edge attibutes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="edge_attrib_tw"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="new_edge_attrib_pb">
              <property name="text">
               <string>New attrib</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="del_edge_attrib_pb">
              <property name="text">
               <string>Del attrib</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="save_edge_pb">
              <property name="text">
               <string>Save edge</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
